portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/DataSet.path

```diff
@@ -131,6 +131,16 @@
 	<td>//tr[contains(.,'${key_fieldName}')]//button[contains(@class,'dropdown-toggle')]//*[contains(@class,'lexicon-icon-ellipsis-v')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>FIELD_COLUMN_NAME</td>
+	<td>//tr//th[contains(.,'${key_type}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>EDIT_CELL_RENDERER</td>
+	<td>//div[contains(@class,'form-group') and contains(.,'${key_type}')]//div[@class="dropdown"]</td>
+	<td></td>
+</tr>
```

Comment:
Can we use a generic dropdown instead of creating a new one?

Reply:
Done!